src/components/ui_parts/card/phoneCard.php

```diff
@@ -0,0 +1,21 @@
+<?php /*
+  概要: スマホ用Cardパーツ
+    title: string
+    description: string
+    image: string
+*/ ?>
+
+<div class="mx-4 rounded-2xl border flex flex-col gap-4 items-center overflow-hidden ">
+  <img src="<?php echo $image ?>" class="h-[280px] w-full">
```

Comment:
[imo]
縦横比維持のほうが良いかも！

```suggestion
  <img src="<?php echo $image ?>" class="aspect-[7/4] w-full">
```